docs/_includes/usage/cf-layout/usage.md

```diff
@@ -1290,208 +1290,181 @@ height of the next parent using `position: relative;`. This means that the
 
 
 ## Featured content module
-{% comment %}
-TODO: Revist the naming of these three modules. The layouts are more about
-positioning the image and less about the type of content they host.
-{% endcomment %}
 
-Featured content module, like a hero, consists of headline/text/optional call
-to action along with a visual. It is intended to be used in a main content
-column next to a sidebar.
+Featured content modules, like a hero, consist of headline and description,
+an optional call to action, and a static or video visual.
+They are intended to be used in a main content column next to a sidebar.
 
 Text is full width & displayed above the visual in the default/mobile view.
-At larger screen sizes, the text occupies a fixed portion of the screen (equal
-to the width of 5 of 12 columns at `701px` & 3 of 12 columns at `901px` for
-desktop). The visual occupies the remaining space. The visual should be
-`640x360` (16x9 ratio) and resize to fit the height of the Featured Content
-Module with a static width and is anchored left when it becomes too wide for
-the available space.
-
-<div class="block block__border block__flush-sides">
-    <section class="o-featured-content-module">
-        <div class="o-featured-content-module_text">
-            <div class="a-heading a-heading__icon">
-                {% include icons/speech-bubble.svg %}
-                Featured
-            </div>
-            <h2>Feature title</h2>
-            <p>
-                Lorem ipsum dolor sit amet, ei ius adhuc inani iudico, labitur
-                instructior ex pri. Cu pri inani constituto, cum aeque noster
-                commodo cu.
-            </p>
-            <a class="a-link
-                      a-link__jump
-                      a-link__icon-after-text">
-                <span class="a-link_text">Read more about the feature</span>
-                {% include icons/right.svg %}
-            </a>
-        </div>
-        <div class="o-featured-content-module_visual">
-            <img class="o-featured-content-module_img"
-                 src="http://placekitten.com/g/540/270">
-        </div>
-    </section>
-</div>
+At larger screen sizes, the image moves to the right of the text,
+occupying a fixed width of 270px (equal to 3 of 12 columns at max page width).
+When creating a static image for the visual, it should be 1076px × 606px
+(a 16:9 aspect ratio), which is 2x the rendered with at the maximum size
```

Comment:
`which is 2x the rendered with` … should be `width` not `with`.

Reply:
Good catch; thanks! Fixed in 43458fae.